test/remote/gateways/remote_global_collect_test.rb

```diff
@@ -55,18 +55,17 @@ def test_successful_purchase_with_more_options
     assert_equal 'Succeeded', response.message
   end
 
-  # When approval is required, purchase will not make a capture request
+  # When approval is required, `purchase` will also make a `capture` request
```

Comment:
Yay for comments! Could you perhaps add something in here that states that the `purchase` request will also send an explicit `capture` request to the gateway?

Reply:
@leila-alderman Yep! I updated this comment to be more explicit, lemme know your thoughts 🙇